### ISSUE TITLE
Please publish new version using new eslint peerDependencies

### ISSUE THREAD

**Issue by loynoir** [JoshuaKGoldberg/eslint-plugin-expect-type#42]
### 🐛 Bug Report

- `eslint-plugin-expect-type` version: 0.0.7
- ESLint version: X.X.X
- Node version: X.X.X

#### Actual Behavior

https://github.com/JoshuaKGoldberg/eslint-plugin-expect-type/blob/a1348e1576216e21afc89f6609cae85582291a71/package.json#L45

```txt
└─┬ eslint-plugin-expect-type                                                                                    
  └── ✕ unmet peer eslint@"^5.0.0 || ^6.0.0": found 8.11.0
```

**Comment by JoshuaKGoldberg** [comment]
Two major versions behind! Absolutely yes, will do.

**Comment by JoshuaKGoldberg** [comment]
While I'm here, I'll generally bump the package dependency versions. This still depends on `@typescript-eslint/experimental-utils` rather than `@typescript-eslint/utils`.

**Comment by JoshuaKGoldberg** [comment]
@loynoir @danvk https://www.npmjs.com/package/eslint-plugin-expect-type/v/0.1.0 should be released with support for `"eslint": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0"`. Please let me know if there's anything else to be done!
